cmd/utils/flags.go

```diff
@@ -300,10 +300,16 @@ var (
 		Usage:    "Scheme to use for storing ethereum state ('hash' or 'path')",
 		Category: flags.StateCategory,
 	}
-	PathDBSyncFlag = &cli.BoolFlag{
-		Name:     "pathdb.sync",
-		Usage:    "sync flush nodes cache to disk in path schema",
-		Value:    false,
+	PathDBNodeBufferTypeFlag = &cli.StringFlag{
+		Name:     "pathdb.nodebuffer",
+		Usage:    "Type of trienodebuffer to cache trie nodes in disklayer('list', 'sync', or 'async')",
```

Comment:
Shall leave the default as `async`? Since it might not be a common demand to get the withdraw proof. If it introduces overhead and uncertainty, I think it's better to use `async` for default.

Reply:
If users change the mode (e.g. from async to list) and restart, will it take effect?